fast_response/scripts/run_track_followup.py

```diff
@@ -24,12 +24,15 @@
                     help="list of events to exclude from this analysis. "
                     "such as HESE events that contributed to the trigger."
                     "Example --alert_id  127853:67093193,128290:6888376")
+parser.add_argument('--suffix', type=str, default='A',
```

Comment:
Thank you thank you thank you for this. For when this is run automatically with a listener, we probably need to add logic to the listener to identify when these events happen. We can save this for a future PR though

Reply:
good point, we can definitely do that (maybe in the listener script itself)